tests/unit_tests/test_rating_service.py

```diff
@@ -343,3 +343,20 @@ async def test_game_rating_error_handled(rating_service, game_info, bad_game_inf
     service._logger.warning.assert_called()
     # second game: results have been saved.
     service._persist_rating_changes.assert_called_once()
+
+
+async def test_game_update_empty_resultset_raises_exception(semiinitialized_service):
+    service = semiinitialized_service
+    game_id = 1
```

Comment:
You want to set this to something for which there are no game_player_stats entries.